build-scripts/Dockerfile.linux-musl-arm64

```diff
@@ -16,7 +16,6 @@ RUN --mount=type=tmpfs,target=${BUILD_DIR} cp -R /src/. ${BUILD_DIR}/ \
     && cargo build --features monitoring_prom,slog_json --release --workspace --target ${TARGET} --bin stacks-node --bin stacks-inspect --bin clarity-cli --bin blockstack-cli \
     && mkdir -p /out \
     && cp -R ${BUILD_DIR}/target/${TARGET}/release/. /out
-
+#  --bin stacks-node --bin stacks-inspect --bin clarity-cli --bin blockstack-cli
```

Comment:
I think this commented line can be deleted.

Reply:
removed